tt_metal/hw/inc/blackhole/dev_mem_map.h

```diff
@@ -55,7 +55,7 @@
 #define MEM_L1_BARRIER 12
 #define MEM_MAILBOX_BASE 16
 // Magic size must be big enough to hold dev_msgs_t.  static_asserts will fire if this is too small
-#define MEM_MAILBOX_SIZE (5 * 4 * 512 + 4 * 32 + 1600 + 160)
+#define MEM_MAILBOX_SIZE 12256
```

Comment:
NIT: I liked how we previously showed the explicit calculation of the mem mailbox size (made my life easier when playing around with these data structures). Any way we can go back to that?